docroot/themes/custom/uids_base/scss/components/page-layouts.scss

```diff
@@ -24,6 +24,157 @@
 .page__container.page__container--full {
   max-width: 100%;
 }
+html {
```

Comment:
@dnkemper Do we want to move this to a separate component file and library?  I don't think I would know to look in the `page-layouts.scss` to find the back to top link.